manager/controlapi/service.go

```diff
@@ -150,6 +151,17 @@ func validateContainerSpec(taskSpec api.TaskSpec) error {
 		return grpc.Errorf(codes.InvalidArgument, "ContainerSpec: %q is not a valid repository/tag", container.Image)
 	}
 
+	for _, host := range container.Hosts {
+		arr := strings.Split(host, ":")
```

Comment:
Oh, I have noticed that before. While the code was from https://github.com/docker/docker/blob/master/opts/hosts.go#L153-L165, and in the comments of func `ValidateExtraHost`, it mentions that ipv6 is OK as well.